dts/riscv/it8xxx2.dtsi

```diff
@@ -624,5 +624,12 @@
 			label = "I2C_5";
 			port-num = <5>;
 		};
+
+		ecpm: clock-controller@f01e00 {
+			compatible = "ite,it8xxx2-ecpm";
+			reg = <0x00F01E00 0x11>;
```

Comment:
Can we use lower-case hex consistently? It is easier to read.

Reply:
Done